src/pokedex.c

```diff
@@ -4522,37 +4537,37 @@ static void UnusedPrintDecimalNum(u8 windowId, u16 b, u8 left, u8 top)
     PrintInfoSubMenuText(windowId, str, left, top);
 }
 
-static void PrintFootprint(u8 windowId, u16 dexNum)
+static void DrawFootprint(u8 windowId, u16 dexNum)
 {
-    u8 image[32 * 4] = {0};
-    const u8 * r12 = gMonFootprintTable[NationalPokedexNumToSpecies(dexNum)];
-    u32 i, j, r5 = 0;
+    u8 footprint[32 * 4] = {0};
+    const u8 * footprintGfx = gMonFootprintTable[NationalPokedexNumToSpecies(dexNum)];
+    u32 i, j, tileIdx = 0;
```

Comment:
I think we should leave this as is because `footprint` isn't zeroed out by default which is undefined behaviour and changing the types of `tileIdx`, `i`, and `j` to `u32` is an optimization to remove unnecessary shift instructions in the assembly.

Reply:
Alright